polygeist/tools/cgeist/Test/Verification/sycl/sycl_extern_func.cpp

```diff
@@ -8,6 +8,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+// clang-format off
```

Comment:
`llvm/polygeist/tools/cgeist/Test/.clang-format` has `DisableFormat: true`. Why this change is needed?

Reply:
My vscode format files when I save them automatically and causes reformatting. The change is harmless, not lit test need formatting.